plugin/hotswap-agent-spring-plugin/src/main/java/org/hotswap/agent/plugin/spring/transformers/support/InitMethodEnhance.java

```diff
@@ -0,0 +1,21 @@
+package org.hotswap.agent.plugin.spring.transformers.support;
+
+public class InitMethodEnhance {
+
+    public static String catchException(String objectName, String loggerName, String exceptionName, String from,
+        boolean hasReturnValue) {
+        return "{"
```

Comment:
pls. make the generated code intent correctly.

Reply:
done

Reply:
pls. indent the code in else clause too.

Reply:
pls. indent the code in else clause too.

Reply:
check again？